### DISCUSSION TITLE
Keybinding to suspend PC

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#5897] [Q&A] [answered]
I want to add keybinding to suspend my PC(running linux mint)
the basic command is pm-suspend. But it requires superuser permission and passsword. Any other way to achieve this task?

**Comment by orestisfl** [comment, marked as answer]
Try `systemctl suspend`

**Reply by ghost** [reply]
oh i see It is "systemctl" not "systemct1".
thank you it is working.
guys use system `systemctl suspend`
